solvers/mosek/mosekbackend.cc

```diff
@@ -535,13 +574,22 @@ void MosekBackend::SetBasis(SolutionBasis basis) {
   ConStatii(constt);
 }
 
+void MosekBackend::AddPrimalDualStart(Solution sol)
+{
+  auto mv = GetValuePresolver().PresolveSolution(
+        { sol.primal, sol.dual } );
+  auto x0 = mv.GetVarValues()();
+  auto pi0 = mv.GetConValues()(CG_Linear);
+  MOSEK_CCALL(MSK_putxx(lp(), MSK_SOL_ITR, (MSKrealt *)&x0));
```

Comment:
x0 and pi0 are std::vector's, so &x0 is the address of the vector object. To access the data pointer, use x0.data()

Reply:
Fixed. Thanks